src/indexes.ts

```diff
@@ -568,6 +569,45 @@ class Index<T = Record<string, any>> {
     return task
   }
 
+  ///
+  /// PAGINATION SETTINGS
+  ///
+
+  /**
+   * Get the pagination settings
+   * @memberof Index
+   * @method getPagination
+   * @returns {Promise<object>} Promise containing object of pagination settings
+   */
+  async getPagination(): Promise<object> {
```

Comment:
Please consider creating a structure like `PaginationSettings` containing the fields.

Reply:
> Please consider creating a structure like `PaginationSettings` containing the fields.

It exists here https://github.com/meilisearch/meilisearch-js/blob/main/src/types/types.ts#L188

Reply:
Oke, could you return that instead of `Object` ?